network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/AbstractLoadingLimits.java

```diff
@@ -72,9 +72,9 @@ public T setPermanentLimit(double permanentLimit) {
     }
 
     @Override
-    public Collection<LoadingLimits.TemporaryLimit> getTemporaryLimits() {
+    public Collection<TemporaryLimit> getTemporaryLimits() {
         return attributes.getTemporaryLimits() == null ? Collections.emptyList()
-                                                       : attributes.getTemporaryLimits().values().stream().map(TemporaryLimitImpl::new).collect(Collectors.toUnmodifiableList());
+            : attributes.getTemporaryLimits().values().stream().sorted().map(TemporaryLimitImpl::new).collect(Collectors.toUnmodifiableList());
```

Comment:
Are you sure it cannot be null anymore?

Reply:
which object are you talking to ?

Reply:
attributes.getTemporaryLimits()

Reply:
oups sorry I didn't read the code correctly

Reply:
.sorted() is useless?

Reply:
With a TreeMap values are already sorted, this is the purpose of a TreeMap vs List

Reply:
yes of course, this is always sorted, but now with the desc order.
this file should not have been changed.

Reply:
Ok sorry I see, the sorting of TemporaryLimit is the invert of the TreeMap one